cloudigrade/api/util.py

```diff
@@ -771,3 +773,52 @@ def process_instance_event(event):
             run.save()
             runs.append(run)
         calculate_max_concurrent_usage_from_runs(runs)
+
+
+def save_instance_type_definitions(definitions, cloud_type=AWS_PROVIDER_STRING):
```

Comment:
I like the decision to move this function.

I think `cloud_type` should become a required argument without a default, though, since AWS shouldn't be assumed as the primary caller, especially since we'll eventually add more clouds after Azure. I'm sure this function will evolve a lot more by the time we add GCP, but I think it sets a good precedent to *start* making the cloud types more explicit everywhere.

Reply:
I was on the fence with this, I appreciate the direction.